tests/unit_tests/test_wave_06.py

```diff
@@ -110,16 +107,13 @@ def test_swap_best_by_category():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
-    # Assertions should check:
-    # - That the results is truthy
-    # - That tai and jesse's inventories are the correct length
-    # - That all the correct items are in tai and jesse's inventories, including the items which were swapped from one vendor to the other
-
-@pytest.mark.skip
+    assert result == True
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+    assert item_f in tai.inventory
+    assert item_c in jesse.inventory
```

Comment:
👍 These two assertions check that `item_c` and `item_f` were swapped.

We should also check that the inventory of the two vendors were not changed beyond swapping the best of a category. That means we should also check that Tai still has `item_a` and `item_b` and that Jesse still has `item_d` and `item_e` in their respective inventories. How would you write these assertions?